eppoclient/lrubanditlogger_test.go

```diff
@@ -0,0 +1,69 @@
+package eppoclient
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+func Test_LruBanditLogger_cacheBanditAction(t *testing.T) {
```

Comment:
Added this test [here](https://github.com/Eppo-exp/golang-sdk/pull/65/files#diff-b3e0cbef7a68276439cc6c6bad61ffbac21a441a36f1d7547e9fa56a1655edc0R41).

It doesn't have steps 1 and 6 because it's not possible to log non-bandit event through `LogBanditAction`.